src/euler3d_mappedGrid.f90

```diff
@@ -0,0 +1,679 @@
+MODULE mg
+
+  REAL(kind=8), PARAMETER :: pi=3.1415926535897932385,twopi=2*pi
+  REAL(kind=8), PARAMETER :: rEarth=637120000.0
```

Comment:
@mandli no problem changing the name of the module.

The Earth's radius is not an exact quantity, since the Earth is really an ellipsoid.  The 6371 km is the average radius I used when approximating the Earth as a sphere.

I think the `REAL(kind=8)` forces the quantity to double precision?